walkthroughs/1C-managing-and-protecting-using-api-mgmt/walkthrough.adoc

```diff
@@ -0,0 +1,381 @@
+... Set _Service Accounts Enabled_ to `ON`.
+... Click *Save* to save the changes.
+. Configure client permissions:
+.. Navigate to *Service Account Roles* tab of the client configuration page.
+.. In the _Client Roles_ drop-down list, select `realm-management`.
+.. In the _Available Roles_ pane, select the `manage-clients` list item and assign the role by clicking *Add selected* button.
+. Get the client secret:
+.. Navigate to the *Credentials* tab of the client configuration page.
+.. Note the value in the *Secret* field (it will be referred to as `ZYNC_SECRET`), it will be used in later steps.
+
+[type=verification]
+Have you configured the `{zync-client-id}` client and noted its Client Secret?
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above. If you are still having issues, contact your administrator.
+
+:sectnums!:
+
+// Task resources go here
+[type=taskResource]
+.Task Resources
+****
+* link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.6/html/using_the_developer_portal/openid-connect[OpenID Connect Integration, window="_blank"]
+****
+
+
+:sectnums:
+
+[time=5]
+== Configure 3scale API Management
+
+=== Log in to 3scale API Management
+
+. Open the link:https://{user-sanitized-username}-admin.{openshift-app-host}[{3scale-ProductName} Login screen, window="_blank"].
+
+. Enter `{user-sanitized-username}` as username and `password` as password. You will be logged in {3scale-ProductName} Dashboard.
+
+[type=verification]
+Can you see the {3scale-ProductName} Dashboard and navigate the main menu?
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above. If you are still having issues, contact your administrator.
+
+=== Add the Order API to Red Hat 3scale
+
+. From the *Dashboard*, select the *New API* item.
+. Select the *Define Manually* option.
+. Enter `orders-api` as the *Name* and *System name*:
+. Leave the *Description* field empty.
+. Click *Add API* at the bottom of the screen. The Order API service will be created and you will be redirected to the *Overview* page.
+
+=== Configure integration with Orders API
+
+. Configure OpenID Connect as authentication method for the API:
+.. Click *Integration > Configuration* from the side navigation.
+.. Select *edit integration settings* in the top right corner.
+.. In the *Authentication* section at the bottom of the screen select *OpenID Connect*.
+.. Click *Update Service*.
+.. When prompted for confirmation, click *OK*.
+
+. Configure the integration with the Orders API:
+.. If you are not already on the *Configuration* page, click *Integration > Configuration* from the side navigation.
+.. Click *add the base URL of your API and save the configuration.*
+.. In the *Private Base URL* field, enter the URL of the *Order Entry System* from the previous Solution Pattern that should look like this: `https://order-entry-ui-<NAMESPACE>.{openshift-app-host}`.
```

Comment:
Should we use the internal OpenShift URL? It'll be something like `http://order-entry-ui.<NAMESPACE>.svc:8080`.

Reply:
Mm, yep, could totally do that, probably that would be a better practice.